oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/base/InfluxInsertRequest.java

```diff
@@ -68,6 +71,15 @@ public InfluxInsertRequest addFieldAsTag(String fieldName, String tagName) {
         return this;
     }
 
+    public InfluxInsertRequest tags(List<SpanTag> tags) {
+        if (!Objects.isNull(tags) && !tags.isEmpty()) {
+            for (final SpanTag tag : tags) {
+                builder.addField(tag.getKey(), tag.getValue());
```

Comment:
Tag keys could be duplicated as it is in the different spans, what will happen here?